File: PHP/car.php
<?php
required_once("account.php");
class Car {
    public $id;
    public $license;
    public $driver;
    public $passanger;

    public function _construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar(){
        echo "License: $this->license Driver: ".$this->driver->name;
    }
}?> 

File: Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passanger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("License: " + license + " Name Driver: " + driver.name + " Passangers: " + passanger);
    }

    public Integer getPassanger(){
        return passanger;
    }

    public void setPassanger(Integer passanger){
        this.passanger = passanger;
    }
}
